src/main/java/foundation/privacybydesign/sms/smssender/RESTSender.java

```diff
@@ -0,0 +1,44 @@
+package foundation.privacybydesign.sms.smssender;
+
+import foundation.privacybydesign.sms.SMSConfiguration;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class RESTSender extends Sender {
+    private byte[] getBytes(String phone, String message) {
```

Comment:
I found it confusing that this function is after its return type, instead of what its return value contains. Perhaps call this `getURL`?

Reply:
It does not return the URL, it only returns a byte array containing the message in a x-www-form-urlencoded format. The implementing classes send this message as body via either SSH or HTTP(S).